voluncheer/static/admin/js/chat_homepage.js

```diff
@@ -0,0 +1,23 @@
+console.log("Sanity check from chat_homepage.js.");
```

Comment:
Seems this got left in from debugging during development.

Reply:
I think this log really helps with debugging in production for JS related issues. I prefer to leave it in.

Reply:
How so? What is this log line signaling to you on production? You can get this information from the Chrome debugging tools.